src/viam/sdk/registry/registry.cpp

```diff
@@ -92,8 +155,65 @@ const google::protobuf::ServiceDescriptor* ResourceRegistration::service_descrip
     return service_descriptor_;
 }
 
+void register_resources() {
```

Comment:
A bit of a gotcha for Viam developers, we'll now need to add new resources here. It should be hard to miss; `client_to_mock_pipeline` handles `Registry` initialization and so tests will fail if resources aren't registered.